production/template.yml

```diff
@@ -0,0 +1,321 @@
+#
+# Copyright 2017 The Jaeger Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+apiVersion: v1
+kind: Template
+labels:
+  template: jaeger-template
+  jaeger-infra: template
+message: 'A Jaeger service has been created in your project.'
+metadata:
+  name: jaeger-template
+  annotations:
+    description: Jaeger Distributed Tracing Server
+    iconClass: icon-cogs
```

Comment:
I think the icon has to be installed on openshift. There is already https://www.todobackend.com/images/logos/golang.png (all-in-one) is using it. We could use that as an intermediate solution.